sources/operator.1.11.0.pxc-operator.json

```diff
@@ -0,0 +1,246 @@
+            "image_hash": "f109a62eb316732d59dd80ed0e013fc9594cbae601586b94023b8c068f7ced7b",
+            "status": "available",
+            "critical": false
+          }
+        },
+        "haproxy": {
+          "2.4.15": {
+            "image_path": "percona/percona-xtradb-cluster-operator:1.11.0-haproxy",
+            "image_hash": "ebeb5b882394fd2a2c5ab302ecb410745a34bbe472a0846efb17ed8f234cf933",
+            "status": "recommended",
+            "critical": false
+          },
+          "2.3.15": {
+            "image_path": "percona/percona-xtradb-cluster-operator:1.10.0-haproxy-8.0.25",
+            "image_hash": "62479be2a21192a3215f03d3f9541decd5ef1737741245ac33ee439915a15128",
+            "status": "available",
+            "critical": false
+          },
+          "2.3.14": {
+            "image_path": "percona/percona-xtradb-cluster-operator:1.10.0-haproxy",
+            "image_hash": "2f06ac4a0f39b2c0253421c3d024291d5ba19d41e35e633ff6ddcf4ba67fd51a",
+            "status": "available",
+            "critical": false
+          }
+        },
+        "backup": {
+          "8.0.27": {
+            "image_path": "percona/percona-xtradb-cluster-operator:1.11.0-pxc8.0-backup",
+            "image_hash": "37a087035f74c26c67e44dd437f6128b5af419902cd416b17f72f82eaaaf2dad",
+            "status": "recommended",
+            "critical": false
+          },
+          "8.0.25": {
+            "image_path": "percona/percona-xtradb-cluster-operator:1.10.0-pxc8.0.25-backup",
+            "image_hash": "c3991f0959a3b4114d7ff629d9d3cdf0dc200c58443ca8ebb1446d8b1cbe416d",
+            "status": "available",
+            "critical": false
+          },
+          "8.0.23": {
+            "image_path": "percona/percona-xtradb-cluster-operator:1.10.0-pxc8.0-backup",
+            "image_hash": "6ab8efb3804d1e519e49ee10eb46b428a837cfdcee222cc5ae2089cc1dc02a6d",
+            "status": "available",
+            "critical": false
+          },
+          "2.4.26-1": {
+            "image_path": "percona/percona-xtradb-cluster-operator:1.10.0-pxc5.7-backup",
+            "image_hash": "0424bb866cb171bd380c87ad3aa98047af90c1e6e61bf4d81d2c36864cb36e4d",
+            "status": "recommended",
+            "critical": false
+          },
+          "2.4.24-2": {
+            "image_path": "percona/percona-xtradb-cluster-operator:1.10.0-pxc5.7.35-backup",
+            "image_hash": "ac9fcd3078107c6492c687eb98215d4e5daf27a02fb3c78ba4b9e9c01f2078b3",
+            "status": "available",
+            "critical": false
+          },
+          "2.4.24": {
+            "image_path": "percona/percona-xtradb-cluster-operator:1.10.0-pxc5.7-backup",
+            "image_hash": "2ff5992220ba251cf064cc2b4d5929e0fdb963db18e35d6c672f9aacb0be3bed",
+            "status": "available",
+            "critical": false
+          }
+        },
+        "log_collector": {
+          "1.8.11": {
```

Comment:
Do we want to start putting fluentbit version here? @hors

Reply:
lets use old approach (use operator version)

Reply:
ok) it didn't catch it)